### PR TITLE
Fix Publish to PyPI action by switching to poetry

### LINKED ISSUE #36: Fix the release on PyPI action
Use poetry to push to PyPI

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #36 

### Review:
- [x] Code approved by HS
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested by switching temporary on "on PR" event. It should work: it failed because release already exists.

File: .github/workflows/pypi_publish.yml [hunk 1]
```diff
@@ -16,25 +16,16 @@ jobs:
...
-       user: __token__
-       password: ${{ secrets.pypi_password }}
+      poetry-version: "1.4"
+
+   - name: Build and publish to PYPI
+     run: |
+       poetry publish --build
+     env:
+       GITHUB: 1
+       POETRY_PYPI_TOKEN_PYPI: ${{ secrets.PYPI_TOKEN }}
```
Comment (henrikstranneheim) [inline .github/workflows/pypi_publish.yml:31]:
```suggestion
       PYPI_TOKENI: ${{ secrets.PYPI_TOKEN }}
```

**Review by henrikstranneheim — APPROVED** [review]
👍

File: .github/workflows/pypi_publish.yml (hunk 1 above)
Comment (northwestwitch) [reply .github/workflows/pypi_publish.yml:31]: Right, thanks!

**Comment by northwestwitch** [comment]
@henrikstranneheim your suggestion didn't work: 
image

Perhaps the TOKEN var is still set as POETRY_PYPI_TOKEN_PYPI somewhere?

**Comment by henrikstranneheim** [comment]
Ok, maybe that was a keyword in the github api that you cannot change. Let’s just revert and merge

**Comment by northwestwitch** [comment]
It worked! 🥳 

image

**Comment by henrikstranneheim** [comment]
🥳

**Pull request merged**